inductiva/coastal/README.md

```diff
@@ -9,29 +9,55 @@ is performed using the [SWASH](https://swash.sourceforge.io/) simulator.
 **Note:**  Some of the following tools require the installation of extra dependencies for the coastal package with
 `pip install --upgrade "inductiva[coastal_extra]"`.
 
-### Example
+### Example 1: Simulating waves on a synthetic bathymetry
 
-Note: You can follow this example in a notebook or run [this script](/demos/py_scripts/coastal_area.py), that contains all the code
-snippets exhibited here. Don't forget to insert your API key
-(`inductiva.api_key = "your_api_key"`) for the script to work! (You can get one by
-filling this [form](https://docs.google.com/forms/d/e/1FAIpQLSflytIIwzaBE_ZzoRloVm3uTo1OQCH6Cqhw3bhFVnC61s7Wmw/viewform?usp=sf_link))
+You can follow this example in your machine to simulate and visualize the wave propagation in a random bathymetry
```

Comment:
take out in your machine, since it may point to them using their machine to run things.